tasks.rb

```diff
@@ -3265,12 +3265,15 @@ def set_hpxml_site(hpxml_file, hpxml)
   if ['base.xml'].include? hpxml_file
     hpxml.site.fuels = [HPXML::FuelTypeElectricity, HPXML::FuelTypeNaturalGas]
     hpxml.site.site_type = HPXML::SiteTypeSuburban
+    hpxml.site.azimuth_of_front_of_home = 180.0
```

Comment:
Since none of these new BuildResHPXML inputs are used by the HPXMLtoOpenStudio measure, I'd prefer that the sample files not add this extra information. It might make users creating HPXML files think that this information is required/used.

My guess is that you added this so that the BuildResHPXML measure tests pass? If that's the case, I would revert the changes to tasks.rb and instead simply [exclude these elements from being compared](https://github.com/NREL/OpenStudio-HPXML/blob/85ff0bf2565cefa16284a7fc123a7af4cc4a04ff/BuildResidentialHPXML/tests/build_residential_hpxml_test.rb#L211) in the BuildResHPXML measure tests.

Reply:
Ah was not aware of that, I reverted tasks.rb and excluded the 3 fields.